test/RibbonThetaVault.ts

```diff
@@ -428,7 +551,7 @@ function behavesLikeRibbonOptionsVault(params: {
       // Create first option
       firstOptionExpiry = moment(latestTimestamp * 1000)
         .startOf("isoWeek")
-        .add(chainId === CHAINID.ETH_MAINNET ? 1 : 0, "week")
+        // .add(chainId === CHAINID.ETH_MAINNET ? 1 : 0, "week")
```

Comment:
Why did we comment this out?

Reply:
I'm getting the wrong otoken address if I move it one week forward. I might be wrong here, but this might be because of the new blocknumber, so the next option is on the same week, not the following week.

Reply:
We can remove it because it's not used